magellan-library/src/main/java/com/wealthfront/magellan/core/Step.kt

```diff
@@ -65,4 +69,14 @@ public abstract class Step<V : ViewBinding>(
   protected open fun onPause(context: Context, binding: V) {}
 
   protected open fun onHide(context: Context, binding: V) {}
+
+  @CallSuper
```

Comment:
Historically, we've avoided needing `@CallSuper` by making the `override` `final` exposing an "onXyz" method. Is that a pattern we considered here?